packages/eslint-config-khan/.prettierrc.js

```diff
@@ -0,0 +1,11 @@
+/* eslint-disable import/no-commonjs */
```

Comment:
Do we need this file? Do we already have a global prettierrc config for all of wonder-stuff?

Reply:
The wonder-stuff .prettierrc.js imports this.  Right now it's importing the published version.  I'll update it to import this one directly.